test/libsyncengine/reconciliation/platform_inconsistency_checker/testplatforminconsistencycheckerworker.cpp

```diff
@@ -154,36 +165,111 @@ void TestPlatformInconsistencyCheckerWorker::testNameClash() {
 #endif
 }
 
+void TestPlatformInconsistencyCheckerWorker::testNameClashAfterRename() {
+    // Create local files
+    for (const std::vector<std::string> nodes = {{"a1"}, {"A"}}; const auto &n : nodes) {
+        std::ofstream ofs(_tempDir.path() / n);
+        ofs << "Some content.\n";
+        ofs.close();
+    }
+
+    // Set up DB
+    const DbNode dbNodeLower(2, _syncPal->_syncDb->rootNode().nodeId(), Str("a1"), Str("a1"), "la", "ra", defaultTime,
+                             defaultTime, defaultTime, NodeType::NodeTypeDirectory, defaultSize, std::nullopt);
+    const DbNode dbNodeUpper(3, _syncPal->_syncDb->rootNode().nodeId(), Str("A"), Str("A"), "lA", "rA", defaultTime, defaultTime,
+                             defaultTime, NodeType::NodeTypeDirectory, defaultSize, std::nullopt);
+    DbNodeId dbNodeIdLower;
+    DbNodeId dbNodeIdUpper;
+    bool constraintError = false;
+    _syncPal->_syncDb->insertNode(dbNodeLower, dbNodeIdLower, constraintError);
+    _syncPal->_syncDb->insertNode(dbNodeUpper, dbNodeIdUpper, constraintError);
+
+    // Set up remote tree
+    const auto remoteParentNode = _syncPal->getUpdateTree(ReplicaSideRemote)->rootNode();
```

Comment:
`getUpdateTree` is removed in an upcomming PR as it duplicate `updtaeTree`
```suggestion
    const auto remoteParentNode = _syncPal->updateTree(ReplicaSideRemote)->rootNode();
```